presto-main/src/test/java/com/facebook/presto/execution/TestTaskManagerConfig.java

```diff
@@ -72,6 +73,7 @@ public void testExplicitPropertyMappings()
                 .put("task.max-index-memory", "512MB")
                 .put("task.share-index-loading", "true")
                 .put("task.max-partial-aggregation-memory", "32MB")
+                .put("task.max-local-exchange-buffer-size", "8MB")
```

Comment:
It is better to pick some random value like 33MB which is unlikely to ever be a default (clearly I or someone else did not do the same above).